frontend/src/router/routeHandlers.ts

```diff
@@ -29,15 +30,27 @@ import type { RouteLocationNormalized } from 'vue-router';
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
+    let applicationAdmins;
+    let userRolesFetchResult;
     // Requires fetching applications the user administers.
     await asyncWrap(fetchApplications());
-    const userRolesFetchResult = await asyncWrap(
-        fetchUserRoleAssignments(selectedApplication.value?.application_id)
-    );
-    Object.assign(to.meta, { userRoleAssignments: userRolesFetchResult.data });
+    if(selectedApplication.value?.application_id === 1) {
+        applicationAdmins = await asyncWrap(
+            fetchApplicationAdmin(selectedApplication.value?.application_id)
+        )
+    } else if(selectedApplication.value?.application_id === 2) {
```

Comment:
The `else if` (for the case of `2`) is also quite wrong as well.
We will have many applications in FAM, what about other "3" "4" "5" applications etc?
Please rewrite for a better logic.